client/fresheyes-web/src/app/components/form/FormSection.tsx

```diff
@@ -168,36 +173,31 @@ const FormSection = ({ username }: { username: string | null | undefined }) => {
       </div>
 
       <div className='flex flex-col gap-6 overflow-x-scroll max-w-5xl  xl:w-3/4 md:p-4 w-full'>
+        <h2 className=' text-base md:text-xl font-semibold'>Want to see how FreshEyes work ? Try cloning some of these repositories</h2>
         <section className='flex flex-col md:flex-row w-full gap-6'>
-          <ClickableOptions
-            owner={"bitcoin"}
-            repo={"bitcoin"}
-            pull_number={8149}
-            onClickOption={() => processPullRequest(true, { owner: "bitcoin", repo: "bitcoin", pull_number: 8149 })}
-            setFormValues={setFormValues}
-          />
+          <ClickableOptions owner={"bitcoin"} repo={"bitcoin"} pull_number={8149} setFormValues={setFormValues} title={"Segregated witness"} />
           <ClickableOptions
             owner={"bitcoindevkit"}
             repo={"bdk"}
             pull_number={593}
-            onClickOption={() => processPullRequest(true, { owner: "bitcoindevkit", repo: "bdk", pull_number: 593 })}
             setFormValues={setFormValues}
+            title={"Add support for Taproot and tr() descriptors"}
           />
         </section>
         <section className='flex flex-col md:flex-row w-full gap-6'>
           <ClickableOptions
             owner={"lightningnetwork"}
             repo={"lnd"}
             pull_number={2022}
-            onClickOption={() => processPullRequest(true, { owner: "lightningnetwork", repo: "lnd", pull_number: 2022 })}
             setFormValues={setFormValues}
+            title={"htlcswitch: hodl invoice"}
```

Comment:
```suggestion
            title={"Htlcswitch: hodl invoice"}
```
Let's be consistent with the alphabet casing

Reply:
I'm making everything here to be in uppercase but corrections are duly noted